CMakeLists.txt

```diff
@@ -20,6 +20,10 @@ SET_PROPERTY(GLOBAL PROPERTY USE_FOLDERS ON)
 
 ########### Project name ###########
 PROJECT(Stellarium C CXX)
+
+SET(STELLARIUM_RELEASE_BUILD 0 CACHE BOOL "Set 1 to build as an official release (0 for development snapshots).")
+SET(STELLARIUM_RELEASE_NUMBER 0)
+
 ########### Detect Qt version
 SET(ENABLE_QT6 1 CACHE BOOL "Whether to try building with Qt6. If Qt6 is not found, Qt5 will be used.")
```

Comment:
In QtCreator, is there a way to detect/analyze the selected "Kit"? When I select Qt5.12 and run cmake on a fresh clone, it happily finds the Qt6.4 installed in parallel until I set this variable manually.

Reply:
Hmm... I don't have described issue in my QtCreator